oauth2/__init__.py

```diff
@@ -119,6 +119,11 @@ def dispatch(self, request, environ):
             response = self.response_class()
             response.add_header("Content-Type", "text/plain")
```

Comment:
As the response body contains JSON now, please change the `Content-Type` to `application/json`.